### PR TITLE
Feature paladin 2.4 fixes broken build

### PR DESCRIPTION
**Pull request by adamfowleruk**

### CODE REVIEW

File: haiku-apps/paladin/paladin-2.4.recipe [hunk 1]
```diff
@@ -25,7 +25,11 @@ COPYRIGHT="2001-2010 DarkWyrm
 LICENSE="MIT"
 REVISION="1"
 SOURCE_URI="https://github.com/adamfowleruk/Paladin/archive/v$portVersion.tar.gz"
+<<<<<<< HEAD:haiku-apps/paladin/paladin-2.4.recipe
```
Comment (diversys) [inline haiku-apps/paladin/paladin-2.4.recipe]: This shouldn't be here.

File: haiku-apps/paladin/paladin-2.4.recipe [hunk 2]
```diff
@@ -25,7 +25,11 @@ COPYRIGHT="2001-2010 DarkWyrm
...
+CHECKSUM_SHA256="12410bb9948e81eedd25c6f114dd6a80ce909774b3c460a215e6adccdfd94bd6"
+=======
```
Comment (diversys) [inline haiku-apps/paladin/paladin-2.4.recipe]: same here

File: haiku-apps/paladin/paladin-2.4.recipe [hunk 3]
```diff
@@ -25,7 +25,11 @@ COPYRIGHT="2001-2010 DarkWyrm
...
 CHECKSUM_SHA256="943cda99d073cf3072475307b56bc93a2a0e9d002dce389317396c419077c6fe"
+>>>>>>> aafd96059b5f807a4829792d6506fca5d4f4dfa1:haiku-apps/paladin/paladin-2.3.recipe
```
Comment (diversys) [inline haiku-apps/paladin/paladin-2.4.recipe]: and here.

**Comment by adamfowleruk** [comment]
Sorry about that. I sorted a merge conflict but git didn't warn me that file itself had changes, just that it was new. Fixed now.

**Pull request merged**